### DISCUSSION TITLE
What does the "value" on color key mean?

### DISCUSSION THREAD

**Discussion by candacelei** [reactome/ReactomeGSA#40] [Q&A] [answered]
Hi, 
I specified a couple of reactome pathways and tried to explore how they differ across clusters in my scRNA-seq data. I got a heatmap by running 
> plot_gsva_heatmap(gsva_result, pathway_id = c("R-HSA-216083","R-HSA-2022090","R-HSA-1442490"),margins = c(6,30))

there's a "value" on the color key with red indicating a lower value and blue indicating a higher value. What does the "value" mean here? 

Thank you for your attention!

**Comment by jgriss** [comment, marked as answer]
Hi @candacelei,

Thanks a lot for your interest in ReactomeGSA!

The values are the pathway-level expression values as estimated by the `ssGSEA` approach. We use the implementation of the `GSVA` Bioconductor package. You can find all the details about the approach [here](http://bioconductor.org/packages/release/bioc/vignettes/GSVA/inst/doc/GSVA.html).

Short version: It's a pathway analysis approach, where the gene- / protein-level abundance values are mapped to the pathway level. For ssRNA-seq data, you can then use this information to detect pathway-level differences instead of only working on the gene level.

Does that help?

Kind regards,
Johannes
